### PR TITLE
added Login, logout and signup pages

### PR DESCRIPTION
**Pull request by omjadhav85**
image
image
image

### CODE REVIEW

**Review by dubeyaditya29 — COMMENTED** [review]
overall code looks pretty clean. Simple and clear UI. Looks Good to merge

File: routes/login.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link rel="stylesheet" href="../styles/login.css" />
+    <title>Sneakoholics | Login</title>
+  </head>
+  <body>
+    <nav class="navbar fixed">
```
Comment (Shivanshsharma01) [inline routes/login.html]: It's good to wrap the main nav in a "header" tag.

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can implement suggestion

File: routes/login.html (hunk 1 above)
Comment (omjadhav85) [reply routes/login.html]: Thanks for the suggestion Shivansh...made the changes.

**Comment by omjadhav85** [comment]
> overall code looks pretty clean. Simple and clear UI. Looks Good to merge

Thanks for reviewing Aditya!

**Pull request merged**